packages/backEnd/src/server.ts

```diff
@@ -30,45 +23,49 @@ process.on("SIGINT", (err) => {
 
 connect();
 
-const server: Express = express();
-server.use(express.urlencoded({ limit: "50mb", extended: true }));
-server.use(express.json({ limit: "50mb" }));
-server.use(compression());
-server.use(express.static("avatar"));
-server.use(
-  cors({
-    methods: ["GET, POST, PUT, DELETE, OPTIONS"],
-    credentials: true,
-    origin,
-    allowedHeaders: [
-      "Origin, X-Requested-With, Content-Type, Accept, Authorization",
-    ],
-  })
-);
+export class SampleServer extends Server {
+  constructor() {
+    super(process.env.NODE_ENV === "development"); // setting showLogs to true
+    this.app.use(json());
+    this.app.use(urlencoded({ extended: true }));
+    this.app.use(express.urlencoded({ limit: "50mb", extended: true }));
+    this.app.use(express.json({ limit: "50mb" }));
+    this.app.use(compression());
+    this.app.use(express.static("avatar"));
+    this.app.use(
+      cors({
+        methods: ["GET, POST, PUT, DELETE, OPTIONS"],
+        credentials: true,
+        origin,
+        allowedHeaders: [
+          "Origin, X-Requested-With, Content-Type, Accept, Authorization",
+        ],
+      })
+    );
+    this.app.use(helemt());
```

Comment:
There shouldn't be helmet? If yes, in import is mistake too ;)

Reply:
I'll check it.